cfme/automate/buttons.py

```diff
@@ -113,7 +114,8 @@ class attribute(ParametrizedView):  # noqa
             def all(cls, browser):
                 return [(i,) for i in range(1, 6)]
 
-        # TODO: Role Access
+        role_show = BootstrapSelect(id="visibility_typ")
+        roles = RolesSelector(locator="//label[contains(text(),'User Roles')]/../div/table")
```

Comment:
I would suggest using a ParametrizedLocator here instead of having to form and pass an entire locator for a highly specific widget. But will accept as-is.